adapters/rest-adapter-generator/src/test/java/uk/gov/justice/services/adapters/rest/generator/RestAdapterGenerator_CodeStructureTest.java

```diff
@@ -374,9 +378,9 @@ public void shouldGenerateResourceClassImplementingInterface() throws Exception
     }
 
     @Test
-    public void shouldGenerateResourceClassContainingAdapterAnnotation() throws Exception {
+    public void shouldGenerateResourceClassContainingCommandApiAdapterAnnotation() throws Exception {
         generator.run(
-                restRamlWithDefaults().with(
+                restRamlWithCommandApiDefaults().with(
                         resource("/some/path")
                                 .with(action(POST, "application/vnd.ctx.commands.default+json"))
```

Comment:
These all need to be updated to be singular once the main 'convert pillar names to singular term' is merged to keep it consistent.